cassandra-analytics-core/src/main/java/org/apache/cassandra/spark/bulkwriter/BulkWriterContext.java

```diff
@@ -26,6 +26,8 @@
 
 public interface BulkWriterContext extends Serializable
 {
+    BulkSparkConf conf();
```

Comment:
this feels out of place. The `JobInfo` seems to be the abstraction to expose the configured values for the context, but now we have both the abstraction and the direct values in the same class. Why have the abstraction in the first place if we are just going to expose the raw configuration here?